napari_crop/_function.py

```diff
@@ -16,6 +17,7 @@ def napari_experimental_provide_function():
 
 @register_function(menu="Utilities > Crop region(s)")
 def crop_region(
+    viewer: 'napari.viewer.Viewer',
     layer: napari.layers.Layer,
     shapes_layer: napari.layers.Shapes,
 ) -> List[LayerDataTuple]:
```

Comment:
```suggestion
def crop_region(
    layer: napari.layers.Layer,
    shapes_layer: napari.layers.Shapes,
    viewer: 'napari.viewer.Viewer' = None, 
) -> List[LayerDataTuple]:
```
This should fix backwards compatibility